Riot/Modules/Settings/DeactivateAccount/DeactivateAccountViewController.m

```diff
@@ -126,6 +125,12 @@ - (void)userInterfaceThemeDidChange
     [ThemeService.shared.theme applyStyleOnNavigationBar:self.navigationController.navigationBar];
 
     self.activityIndicator.backgroundColor = ThemeService.shared.theme.overlayBackgroundColor;
+
+    self.view.backgroundColor = ThemeService.shared.theme.backgroundColor;
+
+    [self setupStringAttributes];
+
+    [self setupViews];
```

Comment:
We can maybe keep a `setupViews` called once in `viewDidLoad` like this:
```
- (void)setupViews {
    // Cancel bar button
    UIBarButtonItem *cancelBarButtonItem = [[UIBarButtonItem alloc] initWithTitle:NSLocalizedStringFromTable(@"cancel", @"Vector", nil) style:UIBarButtonItemStylePlain target:self action:@selector(cancelButtonAction:)];
    self.navigationItem.rightBarButtonItem = cancelBarButtonItem;

    // Deactivate button
    // Adjust button font size for small devices
    self.deactivateAcccountButton.titleLabel.adjustsFontSizeToFitWidth = YES;
    self.deactivateAcccountButton.titleLabel.minimumScaleFactor = 0.5;
    self.deactivateAcccountButton.titleLabel.baselineAdjustment = UIBaselineAdjustmentAlignCenters;
    
    self.deactivateAcccountButton.layer.masksToBounds = YES;
}
```
Rename:
- `setupStringAttributes` to `updateStringAttributes`
- `setupDeactivateAccountInfosLabel` to `updateDeactivateAccountInfosLabel`
- `setupForgetMessagesInfoLabel` to `updateForgetMessagesInfoLabel`

Call above methods in `userInterfaceThemeDidChange` and add other missing lines with `ThemeService.shared.theme...` in it.